apps/emqx_resource/src/schema/emqx_resource_schema.erl

```diff
@@ -60,7 +59,7 @@ fields("creation_opts") ->
         {auto_restart_interval, fun auto_restart_interval/1},
         {query_mode, fun query_mode/1},
         {request_timeout, fun request_timeout/1},
-        {async_inflight_window, fun async_inflight_window/1},
+        {inflight_window, fun inflight_window/1},
```

Comment:
Just a reminder to update this point: https://github.com/kjellwinblad/emqx/blob/27b84453371fe99c3a8b0e2e0e0aaed274b1d31c/apps/emqx_resource/src/emqx_resource_buffer_worker.erl#L196

Reply:
Not sure if I understand what you mean by "Just a reminder" but I have added a commit to this PR that renames all occurrences of `async_inflight_window` to `inflight_window`.

Reply:
Sorry, I think I was seeing just one of the commits.

Maybe it's worth adding an entry to keep track of when this field was renamed, yes.